tests/e2e/test_all_movies_page.py

```diff
@@ -1 +1,13 @@
 # TODO: Feature 1
+from flask import Flask
+from flask.testing import FlaskClient
+
+def test_get_movies_no_data(test_app: FlaskClient): 
+    response = test_app.get('/movies/new')
```

Comment:
why are you testing `/movies/new`?
Shouldn't it be:
```suggestion
    response = test_app.get('/movies')
```

Reply:
Yeah, that was my bad on that. Fixed though!